marcdown_project/marcdown/api_views/user.py

```diff
@@ -0,0 +1,63 @@
+from django.http import JsonResponse
+from rest_framework import viewsets
+from rest_framework.renderers import JSONRenderer
+
+from rest_framework.decorators import action
+
+from django.contrib.auth.models import User
+from marcdown.serializers import ProfileSerializer
+
+class UserViewSet(viewsets.ViewSet):
+    # get
+    def list(self, request):
+        '''
+        Gives all the information pertaining to the authenticated user
+        '''
+        user = request.user
+        if user.is_authenticated:
+            serializer = ProfileSerializer(user.profile)
+            return JsonResponse(serializer.data)
+        else:
+            # TODO: error : unauthenticated
+            pass
+
+    # post
+    def create(self, request):
+        # TODO: register new user using django.contrib.auth
```

Comment:
Since we use django.contrib.auth, no need for this endpoint

Reply:
Whoops, you're absolutely right